controls/raddatetimepicker/getting-started.md

```diff
@@ -10,117 +10,133 @@ position: 3
-_xmlns:telerik="http://schemas.telerik.com/2008/xaml/presentation"_
+#### __[XAML] Example 1: Adding RadDateTimePicker in XAML__
 
-#### __XAML__
 {{region xaml-raddatetimepicke-getting-started_1}}
-	<telerik:RadDateTimePicker VerticalAlignment="Top"/>
+	<telerik:RadDateTimePicker />
 {{endregion}}
 
+>In order to create the __RadDateTimePicker__ control you have to include the following namespace declaration:
+	_xmlns:telerik="http://schemas.telerik.com/2008/xaml/presentation"_
+
 You can also create the __RadDateTimePicker__ in code-behind:
 
-#### __C#__
+#### __[C#] Example 2: Adding RadDateTimePicker in code-behind__
+
 {{region cs-raddatetimepicke-getting-started_2}}
 	RadDateTimePicker radDateTimePicker = new RadDateTimePicker();
 {{endregion}}
 
-#### __VB.NET__
+#### __[VB.NET] Example 2: Adding RadDateTimePicker in code-behind__
+
 {{region vb-raddatetimepicke-getting-started_3}}
 	Dim radDateTimePicker As New RadDateTimePicker()
 {{endregion}}
 
-There are two things to pay attention to in the above code snippet. The first is the declaration of the Telerik.Windows.Controls namespace from Telerik.Windows.Controls.Input assembly and the second is the declaration of the __RadDateTimePicker__ control itself.
+If you run the application now you will see the default __RadDateTimePicker__ control with the default watermark content. Clicking on the control's drop down button will display a date and time representations from which you can select a single value.
 
-If you run your application now you will see the default __RadDateTimePicker__ control.  In the control's input box you will see the default watermark content. Clicking on the control's drop down button will display a date and time representations from which you can select a single value.
+#### __Figure 1: The RadDateTimePicker created by the code in Example 1__
 
->tipBy default, text input is enabled for the __RadDateTimePicker__, because you have the option to enter for instance "3 September" and the tooltip will update its value to suggest 9/3/2010.
+![RadDateTimePicker](images/dateTimePicker_getting_started_1.png)
+
+## Working with Selected Values
 
 You can get the selected value using the following properties:
 
-* __SelectedValue__ - gives you a __DateTime?__ representation of the selected date and time or null if no value has been selected.
+* __SelectedValue__: A __DateTime__ representation of the selected date and time or **null** if no value has been selected.
 
-* __SelectedDate__ - is the __DateTime?__ that corresponds to the user selected date or null, if no value has been selected. 
+* __SelectedDate__: The __Date__ part of the currently selected value. 
 
-* __SelectedTime__ - is of type __TimeSpan?__ and correspond to the selected time representation in the clock view of the control or null if no time has been selected. 
+* __SelectedTime__: The __TimeOfDay__ part of the currently selected value. 
 
 Here is an example of getting these properties:
```

Comment:
In my opinion most of the time we don't need an additional sentence telling what the code snippet shows. This is described in the caption of the snippet. 

If we take this case as an example, we have "Here is an example of getting these properties:" and then on the next line there is "Example 3: Getting the selected values". 

We can leave only the code snippet caption "Example 3: ...."